### PR TITLE
Charter updates

### PR DESCRIPTION
**Pull request by theadactyl**

### CODE REVIEW

**Comment by theadactyl** [comment]
@advaitjain PTAL!

File: sigs/micro/CHARTER.md [hunk 1]
```diff
@@ -14,14 +14,13 @@ Archives of the mailing list will be publicly available.
...
-* [TensorFlow Micro code](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro)
+* [TensorFlow Micro code](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tflite-micro)
 * [SIG Micro monthly meeting notes and agenda](https://goo.gle/tf-micro-notes)
 
 ## Contacts
 
-* Project leads: Neil Tan [@neil-tan](https://github.com/neil-tan), Arm, and Pete Warden [@petewarden](https://github.com/petewarden), Google.
-* For administrative questions, contact Edd Wilder-James
-  [@ewilderj](https://github.com/ewilderj) - ewj at google
+* Project leads: Neil Tan [@neil-tan](https://github.com/neil-tan), Arm, and Pete Warden [@petewarden](https://github.com/petewarden), [Advait Jain] (https://github.com/advaitjain), Google
```
Comment (advaitjain) [inline sigs/micro/CHARTER.md:22]: seems to be an extra space between [Advait Jain] and (https://github.com/advaitjain)

**Comment by advaitjain** [comment]
looks good to me with one small formatting fix.

File: sigs/micro/CHARTER.md (hunk 1 above)
Comment (mihaimaruseac) [inline sigs/micro/CHARTER.md:22]:
```suggestion
* Project leads: Neil Tan [@neil-tan](https://github.com/neil-tan), Arm, and Pete Warden [@petewarden](https://github.com/petewarden), [Advait Jain](https://github.com/advaitjain), Google
```

Comment (mihaimaruseac) [reply sigs/micro/CHARTER.md]: Fixed the typo via 7b349625367ffc7b304a3c2e6183f7f757a4b18f

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**